src/Microsoft.Azure.WebJobs.Host/Retry/RetryResult.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Represents retry settings.
+    /// </summary>
+    public class RetryResult : RetryAttribute
```

Comment:
I don't understand this Type at all. I don't think it's needed.

Reply:
Is it a copy from @jeffhollan 's prototype [here](https://github.com/jeffhollan/retry-design/blob/36d3a4d6704e2dc21b212dae7b9ea6175e87e38b/csharp/EventHubTrigger.cs#L54)?

Reply:
Removed this type for now.